src/device-state.ts

```diff
@@ -41,6 +41,8 @@ import {
 import * as dbFormat from './device-state/db-format';
```

Comment:
Maybe we can merge this commit with with fb0cc12. It seems like tests on that might not pass without some stuff on this commit?

Reply:
This comment refers to the commit 5d55b87, there is no way to make a comment on a commit (that i have found)

Reply:
Done!